### PR TITLE
Update cfs.md

### PR DESCRIPTION
**Pull request by sascala**
<<Replace this line with your Revision Title>>

Summary:
For more details follow http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html.

JIRA issues:

### CODE REVIEW

File: docs/docs/cfs.md [hunk 1]
```diff
@@ -1,19 +1,45 @@
...
 
-In this scenario you need to change the configurations for your DC/OS installation as well as your Mesos Agent configurations. You need to change this line in your [dcos-config.yaml]( https://github.com/dcos/dcos/blob/a7a30779663081198649caecb4d27165836e73ae/gen/dcos-config.yaml#L431) to `MESOS_CGROUPS_ENABLE_CFS=false`. Now you could perform either a DC/OS re-installation or ssh to all Mesos Agent nodes, change the configuration in `/opt/mesosphere/etc/mesos-slave-common` to `MESOS_CGROUPS_ENABLE_CFS=false` and restart the Mesos agent process with `sudo systemctl restart dcos-mesos-slave`.
+The default Mesos configuration uses CPU shares. CFS (Completely Fair Scheduler) strict CPU limitation were introduced as a default in DC/OS a while ago, but until recently this configuration are only respected by the Mesos executor and not by the Docker executor. [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) was fixed in the latest Mesos release, which is also included in DC/OS 1.10.
```
Comment (ichernetsky) [inline docs/docs/cfs.md]: `CPU limitation were introduced` — should be `was`.

Comment (ichernetsky) [inline docs/docs/cfs.md]: `this configuration are only respected` — should be `is`.

File: docs/docs/cfs.md [hunk 2]
```diff
@@ -1,19 +1,45 @@
...
 
-If you are considering changing this configuration, you should also have a look at the [Mesos oversubscription](http://mesos.apache.org/documentation/latest/oversubscription/) feature.
\ No newline at end of file
+Your apps or deployments are likely running slowly because they require more CPU cycles than they are configured to consume.
+
+## Steps to take
+
+### Increase CPU allocation
+
+If you have slow-running Docker applications or deployments due to DC/OS upgrade or configuring `MESOS_CGROUPS_ENABLE_CFS=true`, increase the required CPU amount in your Marathon app definition. Change the `”cpus”` property of your app definition to a higher value and test if this change solves your issues.
```
Comment (ichernetsky) [inline docs/docs/cfs.md]: Please use plain double quotes: `"`, not `”`

File: docs/docs/cfs.md [hunk 3]
```diff
@@ -1,19 +1,45 @@
...
+
+**Note:** If you are considering changing this configuration, consult the [Mesos oversubscription](http://mesos.apache.org/documentation/latest/oversubscription/) documentation for additional considerations and configuration options.
+
+#### DC/OS users
+
+1. Change this line in your [dcos-config.yaml]( https://github.com/dcos/dcos/blob/a7a30779663081198649caecb4d27165836e73ae/gen/dcos-config.yaml#L431) to `MESOS_CGROUPS_ENABLE_CFS=false`.
+
+1. [Reinstall DC/OS](https://dcos.io/docs/1.10/installing/).
+
+1. Alternatively, you can follow the instructions for Non-DC/OS users, below.
```
Comment (ichernetsky) [inline docs/docs/cfs.md]: `Non` -> `non`?

File: docs/docs/cfs.md [hunk 4]
```diff
@@ -1,19 +1,45 @@
...
+
+#### Non-DC/OS users
+
+1. SSH to each Mesos agent nodes.
```
Comment (ichernetsky) [inline docs/docs/cfs.md]: `nodes` -> `node`.

**Comment by sascala** [comment]
Thank you for your careful review, @ichernetsky

**Pull request merged**